websockets/framing.py

```diff
@@ -182,9 +188,9 @@ def check_frame(frame):
     if frame.rsv1 or frame.rsv2 or frame.rsv3:
         raise WebSocketProtocolError("Reserved bits must be 0")
 
-    if frame.opcode in (OP_CONT, OP_TEXT, OP_BINARY):
+    if frame.opcode in [OP_CONT, OP_TEXT, OP_BINARY]:
```

Comment:
I didn't know the performance difference.

It would be faster to create the tuple or list at module level anyway...